### ISSUE TITLE
repeating gradient slices not equal

### ISSUE THREAD

**Issue by ghost** [LeaVerou/conic-gradient#22]
My code looks like this:

``` SCSS
$a: 4%;

background: repeating-conic-gradient(#000 0 .5*$a, transparent 0 $a);
```

Expected result: all slices, whether they're black or transparent should be equal (to `2%` of `360deg`, which is `7.2deg`).

Actual result: they're different ([test](https://codepen.io/thebabydino/pen/BWGmvY)) and the issue is particularly obvious in the `0`/ `360deg` area.

**Comment by LeaVerou** [comment]
Screenshot? This is what I see:

![image](https://cloud.githubusercontent.com/assets/175836/24520765/9c1fe00e-1557-11e7-8d94-373e399300a9.png)

**Comment by ghost** [comment]
Yes, that's what I see too. All slices should be equal, but the transparent one at the top is visibly smaller than the two black ones framing it (in general, the transparent slices look smaller). This causes problems if I have two different elements with such gradients one on top of the other, the top one being rotated by a slice ([live test](https://codepen.io/thebabydino/pen/yMGpmN)).

![screenshot](https://i.imgur.com/KMLVOk6.png)

**Comment by LeaVerou** [comment]
Ah, I see now, thanks!

**Comment by eonj** [comment]
I think this is duplicate of #4, floating point error problem.
